GVFS/GVFS.Platform.Mac/MacFileSystem.cs

```diff
@@ -29,6 +30,24 @@ public override bool IsSocket(string fileName)
             return NativeStat.IsSock(statBuffer.Mode);
         }
 
+        public override bool IsFileSystemSupported(string path, out string error)
+        {
+            string lowerCaseFile = Guid.NewGuid().ToString().ToLower();
+            string upperCaseFile = lowerCaseFile.ToUpper();
+
+            File.Create(Path.Combine(path, lowerCaseFile));
+            if (File.Exists(upperCaseFile))

```

Comment:
Should this check be reversed?  If we create the lower-case file and the upper-case file exists, doesn't that mean the file system is case insensitive?